plugins/flatpak/gs-flatpak.c

```diff
@@ -1301,6 +1320,39 @@ gs_flatpak_app_install_source (GsFlatpak *self, GsApp *app,
 	return TRUE;
 }
 
+static GsApp *
+get_main_app_of_related (GsFlatpak *self,
+			 GsApp *related_app,
+			 GCancellable *cancellable,
+			 GError **error)
+{
+	g_autofree gchar *main_app_name = g_strdup (gs_flatpak_app_get_ref_name (related_app));
+	g_autoptr(FlatpakInstalledRef) ref = NULL;
+	const gchar *app_name;
+	g_auto(GStrv) app_tokens = NULL;
+
+        app_name = gs_flatpak_app_get_main_app (related_app);
+	if (app_name == NULL)
```

Comment:
It looks like you should set the GError here, if this happens for any reason (e.g. maybe with `G_IO_ERROR_NOT_FOUND`)